tests/presses/test_presses.py

```diff
@@ -6,34 +6,34 @@
 from torch import nn
 from transformers import DynamicCache
 
-from kvpress import (
-    BasePress,
+from kvpress.presses.default_presses import (
     ExpectedAttentionPress,
     KnormPress,
     ObservedAttentionPress,
     RandomPress,
     SnapKVPress,
     StreamingLLMPress,
     TOVAPress,
-    ThinKPress,
 )
-
+from kvpress.presses.think_press import ThinKPress
+from kvpress.prunners.default_pruner import DefaultPruner
+from kvpress.scorers.base_scorer import BaseScorer
 from tests.fixtures import unit_test_model, unit_test_model_output_attention  # noqa: F401
 
 
 def test_think_inner_press(unit_test_model):  # noqa: F811
-    press = ThinKPress(compression_ratio=0.5, window_size=2, inner_press=KnormPress(0.5))
+    press = ThinKPress(key_channel_compression_ratio=0.5, window_size=2, inner_press=KnormPress(0.5))
     with press(unit_test_model):
         input_ids = unit_test_model.dummy_inputs["input_ids"]
         unit_test_model(input_ids, past_key_values=DynamicCache()).past_key_values
 
 
 def test_presses_run(unit_test_model):  # noqa: F811
-    for cls in [KnormPress, ExpectedAttentionPress, RandomPress, StreamingLLMPress, SnapKVPress, TOVAPress, ThinKPress]:
+    for cls in [KnormPress, ExpectedAttentionPress, RandomPress, StreamingLLMPress, SnapKVPress, TOVAPress]:
         for compression_ratio in [0.2, 0.4, 0.6, 0.8]:
             press = cls(compression_ratio=compression_ratio)
-            if cls in [SnapKVPress, ThinKPress]:
-                press.window_size = 2
+            if cls in [SnapKVPress]:
```

Comment:
you don't have an error without adding ThinkPress ?

Reply:
I removed `ThinkPress` of the list above b/c of renaming of the `compression_ratio` argument in the press.